extra-scientific/igv/autobuild/build

```diff
@@ -1,5 +1,7 @@
-mkdir -p ${PKGDIR}/usr/share/java
-mv lib/*.jar ${PKGDIR}/usr/share/java
-unzip ${PKGDIR}/usr/share/java/igv.jar org/broad/igv/ui/resources/IGV_64.png
+mkdir -p ${PKGDIR}/usr/share/java/igv
```

Comment:
Missing verbosity and abinfo.

Reply:
Please quote variables.

Reply:
Added verbosity and abinfo for the 3 steps in build script.